res/database/Default/user/scr0llbaer.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <tvtdb>
-	<version value="3" comment="Data entries by scr0llbaer. TODO: Apply references to roles in scripts. TODO: maybe eventually re-introduce 'funny numbers' for production_limit" />
+	<version value="3" comment="Data entries by scr0llbaer. TODO: maybe eventually re-introduce 'funny numbers' for production_limit" />
```

Comment:
I assume you have thoroughly tested the resolution - I did not test-play but only glanced over the changes

Reply:
Double-checked again.
Found one false reference on my part.

And found this issue: https://github.com/TVTower/TVTower/issues/991